packages/superset-ui-demo/storybook/stories/index.js

```diff
@@ -1,3 +1,5 @@
+import 'core-js/stable';
```

Comment:
Are these needed?

Reply:
oops, my bad. that must've gotten carried along from when i was trying to get the storybook to work